src/components/dialogs/create-study-dialog.js

```diff
@@ -167,43 +174,79 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
     const [triggerReset, setTriggerReset] = React.useState(true);
 
     const [isParamsDisplayed, setIsParamsDisplayed] = useState(false);
-
+    const [isParamsCaseFileDisplayed, setIsParamsCaseFileDisplayed] =
+        useState(false);
+    const [isUploadingFileInProgress, setUploadingFileInProgress] =
+        useState(false);
     const handleShowParametersClick = () => {
         setIsParamsDisplayed((oldValue) => !oldValue);
     };
+    const handleShowParametersForCaseFileClick = () => {
+        setIsParamsCaseFileDisplayed((oldValue) => !oldValue);
+    };
 
     const [currentParameters, paramsComponent, resetImportParamsToDefault] =
         useImportExportParams(formatWithParameters);
 
+    const [selectedFile, FileField, selectedFileError, isSelectedFileOk] =
+        useFileValue({
+            label: 'Case',
+            triggerReset,
+            isLoading: isUploadingFileInProgress,
+        });
     //Inits the dialog
     useEffect(() => {
         if (open && providedCase) {
             setStudyName(providedCase.elementName);
             setCaseExist(true);
             dispatch(selectCase(providedCase.elementUuid));
-            getCaseImportParameters(providedCase.elementUuid)
-                .then((result) => {
-                    // sort possible values alphabetically to display select options sorted
-                    result.parameters = result.parameters?.map((p) => {
-                        let sortedPossibleValue = p.possibleValues?.sort(
-                            (a, b) => a.localeCompare(b)
+            handleCaseImportParams(
+                providedCase.elementUuid,
+                setFormatWithParameters
+            );
+        } else if (open && selectedFile && isSelectedFileOk) {
```

Comment:
Why `selectedFile` and `isSelectedFileOk` ?
Just `isSelectedFileOk` ?

Reply:
i changed to open && selectedFile
i used isSelectedFileOk to be more sure